catalog/dags/providers/provider_api_scripts/brooklyn_museum.py

```diff
@@ -101,35 +101,34 @@ def _get_creators(data):
 
     @staticmethod
     def _handle_object_data(data, license_url) -> list[dict]:
-        images = []
-        image_info = data.get("images")
-        if image_info is None:
+        # id_ is used to create the foreign_landing_url, which is a required field
+        if not (image_info := data.get("images")) or not (id_ := data.get("id")):
             return []
 
-        id_ = data.get("id")
-        if id_ is None:
-            return []
+        foreign_landing_url = (
+            f"https://www.brooklynmuseum.org/opencollection/objects/{id_}"
+        )
 
         title = data.get("title", "")
-        foreign_url = f"https://www.brooklynmuseum.org/opencollection/objects/{id_}"
         metadata = BrooklynMuseumDataIngester._get_metadata(data)
         creators = BrooklynMuseumDataIngester._get_creators(data)
 
+        images = []
         for image in image_info:
-            foreign_id = image.get("id")
-            if foreign_id is None:
+            if not (foreign_identifier := image.get("id")):
                 continue
-            image_url = image.get("largest_derivative_url")
-            if image_url is None:
+            if not (image_url := image.get("largest_derivative_url")):
                 continue
             height, width = BrooklynMuseumDataIngester._get_image_sizes(image)
             license_info = get_license_info(license_url=license_url)
+            if not license_info.url:
+                continue
```

Comment:
Does this change potentially conflict with https://github.com/WordPress/openverse/pull/1898? After that PR, would this line be changed to just check that `license_info` isn't None?

Reply:
Considering that the `license_info` refactor is of a higher priority (Wikimedia ingestion is failing without it), this should be rebased onto #1898.